runtime/src/lib.rs

```diff
@@ -211,6 +216,12 @@ pub const UNIT: Balance = 1_000_000_000_000;
 pub const MILLIUNIT: Balance = 1_000_000_000;
 pub const MICROUNIT: Balance = 1_000_000;
 
+// Deposits
+
+pub const fn deposit(items: u32, bytes: u32) -> Balance {
```

Comment:
I believe this is defined in the contracts module as well - can we remove the one there and used this one instead?

Also, not sure it needs to be `pub`, `pub(crate) `probably sufficient.

Reply:
We should ideally use the deposit function part of polkadot constants. But I am happy to amend the code base until have the proper thing in place.

Reply:
This can be `pub (crate)`